test/fixtures/typings/integration/type.d.ts

```diff
@@ -106,6 +106,8 @@ declare module Type {
   type stringLiteralType = "foobar";
   type booleanLiteralType = true;
   type numberLiteralType = 100;
+  type nullLiteralType = null;
+  type neverLiteralType = never;
```

Comment:
https://github.com/Microsoft/TypeScript/blob/master/src/compiler/types.ts#L2932-L2933

TypeScript's TypeFlags does not have `NullLiteralType` and `NeverLiteralType`, so I think that it is better to change this names from `*LiteralType` to `*Type`.